### PR TITLE
login page enhanced #226

### PR DESCRIPTION
**Pull request by suman-tewary**
Login page updated
image

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -296,3 +302,70 @@ div#picker {
     width: 100%;
   }
 }
+.logsignin{
```
Comment (narayan954) [inline src/index.css:305]: provide a comment before the style so anyone can understand which section is the css for. Also logsignin is not a proper naming convention. Use proper classnames, for example refer to the other classnames or [This article](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/)

Comment (narayan954) [reply src/index.css:305]: this applies to the other classname as well below.

File: src/index.css [hunk 2]
```diff
@@ -161,6 +164,9 @@
 .modal__signup {
   display: flex;
   flex-direction: column;
+  display: flex;
```
Comment (narayan954) [inline src/index.css:167]: remove this duplicate property

File: src/index.css [hunk 3]
```diff
@@ -296,3 +302,70 @@ div#picker {
     width: 100%;
   }
 }
+.logsignin{
+  display: flex;
+  width: 100%;
+  justify-content: center;
+}
+button {
```
Comment (narayan954) [inline src/index.css:310]: Avoid giving the pure html elements their property, instead be specific to a class or id as custom components would also get affected (as they render the pure elements themselves ) and you might see abnormal behaviours later. Create a class instead and give class name to those elements like input and button.

**Review by narayan954 — CHANGES_REQUESTED** [review]
Follow the comments I have made and header's design is disturbed by these changes, please fix that or isolate the changes.
headder

**Comment by suman-tewary** [comment]
Please check all  the required changes are done

**Review by narayan954 — APPROVED** [review]
looks awesome to me!

**Comment by narayan954** [comment]
btw can you take a look at the [deepsource analysis](https://deepsource.io/gh/narayan954/dummygram/run/51f14d11-cb06-4a54-9e4d-13534f0f2bf5/javascript/) of this pr? It'd be helpful to you. You can fix issues of [unused variable](https://deepsource.io/gh/narayan954/dummygram/run/51f14d11-cb06-4a54-9e4d-13534f0f2bf5/javascript/JS-0128?listindex=1) [self closing tags](https://deepsource.io/gh/narayan954/dummygram/run/51f14d11-cb06-4a54-9e4d-13534f0f2bf5/javascript/JS-0468?listindex=3) and [class instead of classname for props](https://deepsource.io/gh/narayan954/dummygram/run/51f14d11-cb06-4a54-9e4d-13534f0f2bf5/javascript/JS-0455?listindex=4) issue

**Review by narayan954 — APPROVED** [review]
lgtm!

**Comment by narayan954** [comment]
Thanks for your contribution!! :)

**Pull request merged**